src/parser/core/tests/snapshotTest.js

```diff
@@ -55,9 +55,10 @@ export function tab(analyzer, parser=null) {
  */
 export default function snapshotTest(parserClass, moduleClass, key, propFn = statistic) {
```

Comment:
Where is the function used? I can't find it. Unused functions should be removed.